website/site/content/docs/configuration-options.md

```diff
@@ -133,6 +133,9 @@ For folder collections where users can create new items, the `slug` option speci
 - `{{year}}`: 4-digit year of the file creation date
 - `{{month}}`: 2-digit month of the file creation date
 - `{{day}}`: 2-digit day of the month of the file creation date
+- `{{hour}}`: 2-digit hour of the file creation date
+- `{{minute}}`: 2-digit minute of the month of the file creation date
+- `{{second}}`: 2-digit second of the month of the file creation date
```

Comment:
Typos in these last two lines (extra "of the month" in each).

Reply:
I fixed it!